build.gradle.kts

```diff
@@ -26,6 +26,7 @@ allprojects {
 
     repositories {
         mavenCentral()
+        mavenLocal()
```

Comment:
I presume this means it's using the local Maven Cache? Could we make sure for pulling in TFM it's pulling from Nexus in Maven, so this won't fail if you've not built TFM before... https://nexus-01.core.atlas-media.co.uk/repository/totalfreedom-development/ is the URL you should be able to point Maven at and it'll be able to pull down TFM releases and Release Candidate Builds.

Reply:
thank you, did not know of the nexus repo. will update when home